community/README.md

```diff
@@ -11,6 +11,7 @@ This directory contains plugins contributed by community members.
 #### Path Traversal
 *   [Apache HTTP Server Path traversal and disclosure (CVE-2021-41773) Detector](https://github.com/google/tsunami-security-scanner-plugins/tree/master/community/detectors/apache_http_server_cve_2021_41773)
 *   [Apache Solr RemoteStreaming Arbitrary File Reading Detector](https://github.com/google/tsunami-security-scanner-plugins/tree/master/community/detectors/apache_solr_arbitrary_file_reading)
+*   [Grafana Pre-Auth Arbitrary File Reading vulnerability Detector](https://github.com/google/tsunami-security-scanner-plugins/tree/master/community/detectors/grafana_arbitrary_file_reading)
```

Comment:
Since this vulnerability has a CVE ID assigned, can you please change the detector directory to include such ID?

Reply:
done.